ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/page/v2/.content.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Page component should have a read me like other components. The readme should describe the reasoning behind this new version

Reply:
done